mindsdb/integrations/handlers/chromadb_handler/chromadb_handler.py

```diff
@@ -0,0 +1,368 @@
+            client_settings=Settings(**self._client_config),
+            collection_name=collection_name,
+        )
+
+        return Response(resp_type=RESPONSE_TYPE.OK)
+
+    def query(self, query: ASTNode) -> Response:
+        """
+        Execute a query on the ChromaDB database.
+        """
+        try:
+            self.connect()
+
+            if isinstance(query, Select):
+                return self.query_collection(query)
+
+            elif isinstance(query, CreateTable):
+                return self.run_create_table(query)
+
+            elif isinstance(query, Insert):
+                return self.run_insert(query)
+
+            else:
+                raise NotImplementedError(
+                    f"Unsupported query type {query.__class__.__name__}!"
+                )
+
+        except Exception as e:
+            log.logger.error(f"Error executing query on ChromaDB client, {e}!")
+            return Response(
+                resp_type=RESPONSE_TYPE.ERROR,
+                error_message=f"Error executing query on ChromaDB client, {e}!",
+            )
+
+    def native_query(self, query: str) -> Response:
+        """
+        Execute a native query on the ChromaDB database.
+        """
+
+        # todo fix this - not sure if we need this?
+        try:
+            self.connect()
+            # parse query and extract collection name and any conditions
+            collection_name = extract_collection_name(query)
+            self._collection = self._client.get_collection(collection_name)
+
+        except Exception as e:
+            log.logger.error(
+                f'Error executing native query on ChromaDB client {self._connection_data["chroma_server_host"]}, {e}!'
+            )
+            return Response(
+                resp_type=RESPONSE_TYPE.ERROR,
+                error_message=f'Error executing native query on ChromaDB client {self._connection_data["chroma_server_host"]}, {e}!',
+            )
+
+    def get_tables(self) -> Response:
+        """Get the list of indexes/collections in the vectorDB.
+
+        Returns:
+           Response: The response object.
+        """
+
+        try:
+            self.connect()
+            collections = self._client.list_collections()
+            collections_name = pd.DataFrame(
+                columns=["table_name"],
+                data=[collection.name for collection in collections],
+            )
+        except Exception as e:
+            log.logger.error(
+                f'Error getting tables from ChromaDB client {self._connection_data["chroma_server_host"]}, {e}!'
+            )
+            return Response(
+                resp_type=RESPONSE_TYPE.ERROR,
+                error_message=f'Error getting tables from ChromaDB client {self._connection_data["chroma_server_host"]}, {e}!',
+            )
+
+        return Response(
+            resp_type=RESPONSE_TYPE.TABLE,
+            data_frame=collections_name,
+        )
+
+
+connection_args = OrderedDict(
```

Comment:
Should these (args and example) be here? I can't find usages apart from the import checks.

Perhaps let's move them to some file (e.g. `chromadb_handler/constants.py`?) and mention them in the `README.md` for any interested contributors to be aware that these are the definitions.

Reply:
I copied this from DuckDB handler, its used as a data input validation via UI. 

If param is entered that is not listed here it will raise an error via UI